code/game/objects/items/devices/button.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/money_button
+	name = "red button"
+	desc = "If you press this button, you will get 1000 credits. HOWEVER, one random crewmember will die, which can be you. If you press the button, the button will transfer to someone else."
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "bigred"
+	item_state = "electronic"
+	w_class = WEIGHT_CLASS_TINY
+	origin_tech = "syndicate=2"
+	var/mob/living/last_pressed_person
+
+/obj/item/money_button/attack_self(mob/user)
```

Comment:
maaaybe there should be a cooldown, i can imagine every new recipient immediately pressing it until the crew is reduced to 5 people in the span of a minute. although maybe that would be funnier

Reply:
Ok but it funny (Adding a cooldown of 1 minute)